docs/topics/testing/overview.txt

```diff
@@ -259,6 +259,13 @@ To prevent serialized data from being loaded twice, setting
 :data:`~django.db.models.signals.post_migrate` signal when flushing the test
 database.
 
+.. versionchanged:: 1.10
```

Comment:
Doesn't the first sentence of this section need to be revised, "Any initial data loaded in migrations will only be available in ``TestCase`` tests and not in ``TransactionTestCase`` tests"?